samples/otlptracegrpc/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "sample-app-grpc",
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "main": "index.ts",
+  "scripts": {
+    "start": "node ./app.ts"
```

Comment:
Does this work for you? Node cannot run typescript code directly

Reply:
No, I did not use the node scripts, but yes I should fix this.